src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -471,7 +469,7 @@ protected int engineDoFinal(ByteBuffer src, ByteBuffer dst)
             // Release crypto engine
             engine = null;
             if (encodedKey != null) {
-                Arrays.fill(encodedKey, (byte) 0);
+                Arrays.fill(encodedKey, (byte)0);
```

Comment:
Looks like another unintentional newer->older change.

Reply:
I don't remember an old comment about that, dunno if that was reverted